CorePayments/src/main/java/com/paypal/android/corepayments/PayPalSDKError.kt

```diff
@@ -1,9 +1,11 @@
 package com.paypal.android.corepayments
 
+import androidx.annotation.RestrictTo
+
 /**
  * Class used to describe PayPal errors when they occur.
  */
```

Comment:
Should these docs be suppressed?

Reply:
We actually still need to generate docs for PPCP using dokka, so for now it should be fine. I'm hoping that these get rendered as class level docs. It may be difficult to fully suppress docs for the library-private `constructor` because of the Kotlin syntax.